### ISSUE TITLE
A bug in blender amass

### ISSUE THREAD

**Issue by ChenyangWang95** [openxrlab/xrfeitoria#19]
hi, when I apply the 07_amass.py, error occurs.

```shell
type object 'XRFeitoriaBlenderFactory' has no attribute 'apply_motion_data_to_actor'  
File "E:\Project\xrfeitoria\xrfeitoria\utils\functions\blender_functions.py", line 69
```

**Comment by HaiyiMei** [comment]
Sorry, for that. Please try to use dev_plugins first:

```bash
git clone https://github.com/openxrlab/xrfeitoria.git
cd xrfeitoria
pip install .
python -c "import xrfeitoria as xf; xf.init_blender(replace_plugin=True, dev_plugin=True)"
```

This would be a formal feature when a new version is released.

**Comment by Time-Lord12th** [comment]
> Sorry, for that. Please try to use dev_plugins first:
> 
> ```shell
> git clone https://github.com/openxrlab/xrfeitoria.git
> cd xrfeitoria
> pip install .
> python -c "import xrfeitoria as xf; xf.init_blender(replace_plugin=True, dev_plugin=True)"
> ```
> 
> This would be a formal feature when a new version is released.

Sorry to interrupt, I run your command and add `replace_plugin=True, dev_plugin=True` to blender.07_amass.py. But there's a new error `ImportError: cannot import name 'Dict' from 'xrfeitoria.utils.functions.blender_functions'`. Could you tell me how to solve it? Thanks!
![image](https://github.com/openxrlab/xrfeitoria/assets/53140195/50ad667c-a8a6-41f1-8fe2-8906187440d5)

**Comment by HaiyiMei** [comment]
> > Sorry, for that. Please try to use dev_plugins first:
> > ```shell
> > git clone https://github.com/openxrlab/xrfeitoria.git
> > cd xrfeitoria
> > pip install .
> > python -c "import xrfeitoria as xf; xf.init_blender(replace_plugin=True, dev_plugin=True)"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This would be a formal feature when a new version is released.
> 
> Sorry to interrupt, I run your command and add `replace_plugin=True, dev_plugin=True` to blender.07_amass.py. But there's a new error `ImportError: cannot import name 'Dict' from 'xrfeitoria.utils.functions.blender_functions'`. Could you tell me how to solve it? Thanks! ![image](https://private-user-images.githubusercontent.com/53140195/297810510-50ad667c-a8a6-41f1-8fe2-8906187440d5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2MzM0NzksIm5iZiI6MTcwNTYzMzE3OSwicGF0aCI6Ii81MzE0MDE5NS8yOTc4MTA1MTAtNTBhZDY2N2MtYThhNi00MWYxLThmZTItODkwNjE4NzQ0MGQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE5VDAyNTkzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjZWU4ZWRiOTNjNDRjMzlkYTQ5MDA5ZTYxZmViMGM0NTU4MDVhY2YzZWY3MzRlN2MzMjdhNWZmMWE2ODhlNTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZuAgIy8YLe0-q6dKYcjgfn5CRnCRcSRoTMBaFwwuEBk)

I don't know the exact reason why this happened. But I noticed in the second line, the log says `Reuse existing engine process`. This message suggests that an instance of Blender is already running. To address this, you can try terminating the existing Blender process and then try to run the code again. This may resolve the issue.

XRFeitoria would first search for an existing engine process for efficiency. Once found, it would reuse the engine, and execute the following codes on this existing engine. Normally, this would be fine, but when using the existing engine, the `dev_plugin` and `replace_plugin` are ignored actually, which would cause failure in `replace_plugin`.

**Comment by Time-Lord12th** [comment]
@HaiyiMei You're right. Problem solved, thanks!

**Comment by HaiyiMei** [comment]
Hi @Time-Lord12th @ChenyangWang95, we released `v0.6.0` of XRFeitoria. 

Please update using:
```bash
pip install xrfeitoria -U
```

And you may need to use `xf.init_blender(..., replace_plugin=True)`
